Formula/utopiops.rb

```diff
@@ -0,0 +1,14 @@
+class Utopiops < Formula
+  desc "CLI tool for utopiops.com"
+  homepage "https://www.utopiops.com/"
+  url "https://github.com/utopiops/homebrew-utopiops/releases/download/fresh/utopiops-0.0.3.tar.gz"
+  sha256 "76267f9feff31860ba0d576a09075c1afb7e0e59266ab89037bf4db79301b364"
+
+  def install
+    bin.install "utopiops"
+  end
+
+  test do
+    system "#{bin}/utopiops", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula